tests/test_edge_cases.py

```diff
@@ -1552,7 +1566,7 @@ class MyModel(BaseModel):
     assert dict(m) == {'foo': None}
 
 
-@pytest.mark.xfail(reason='working on V2')
+@pytest.mark.xfail(reason='working on V2 - typevar-handling will be improved after merging generics')
```

Comment:
Hasn't generics been merged? Maybe this needs a rebase / merge?

Reply:
good catch — I think this can probably be tweaked to fix these now. I'll do that and resolve this once it's done.